tests/libvmi/status.go

```diff
@@ -13,15 +13,15 @@ import (
 	"kubevirt.io/kubevirt/pkg/controller"
 )
 
-func GetPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance, namespace string) *k8sv1.Pod {
+func GetPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance, namespace string) (*k8sv1.Pod, error) {
 	virtCli, err := kubecli.GetKubevirtClient()
 	if err != nil {
-		panic(err)
+		return nil, err
```

Comment:
In the e2e tests, it is a convention to panic if the client cannot be created/fetched.
If this errors, there is nothing we can do.

Reply:
Done